site/app/controllers/admin/LateController.php

```diff
@@ -359,8 +366,8 @@ private function parseAndValidateCsv(string $csv_file, array &$data, string $typ
                 return trim($k);
             }, $fields);
 
-            //Each row has three fields
-            if (count($fields) !== 3) {
+            //All types have 3 fields except for exceptions, which can have 3 or 4 rows.
+            if (count($fields) !== 3 && !($type == 'extension' && count($fields) == 4)) {
```

Comment:
It's almost always a good idea to use PHP's strict comparison operators whenever possible.

```suggestion
            if (count($fields) !== 3 && !($type === 'extension' && count($fields) === 4)) {
```